sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -1023,6 +1017,30 @@ export class ApplicationControllerService {
       options?.isChangeRequestSubmission ?? false;
   }
 
+  /**
+   * Execute the temporary validation for full-time submission.
+   * @param firstName students' first name.
+   * @param lastName students' last name.
+   * @throws {UnprocessableEntityException} if full-time submission is not allowed.
+   * @throws {ForbiddenException} if the user is not authorized to submit a full-time application.
+   */
+  private async validateFullTimeSubmission(
+    firstName?: string,
+    lastName?: string,
```

Comment:
What is the reason for keeping last name as optional input?

Reply:
No reason, just fixed.